base_layer/mmr/src/mutable_mmr.rs

```diff
@@ -24,6 +24,7 @@ use crate::{
     backend::ArrayLike,
     common::{leaf_index, n_leaves},
     error::MerkleMountainRangeError,
+    merkle_proof::MerkleProof,
```

Comment:
The idea was not to have this dependency. Only MerkleProof knows about MMR, not the other way round

Reply:
Dependency removed.